modality-probe-debug-collector/src/main.rs

```diff
@@ -0,0 +1,10 @@
+use structopt::StructOpt;
+
+mod cli;
+use cli::{config_from_options, CLIOptions};
+
+fn main() {
+    let opts = CLIOptions::from_args();
+    let config = config_from_options(opts);
+    println!("Running debug collector with configuration: {:#?}", config);
```

Comment:
I don't think I would expect to see this type of output unless some sort of `--verbose` flags was passed.